evaluator/src/main/kotlin/PackageRule.kt

```diff
@@ -54,7 +54,10 @@ open class PackageRule(
     private val licenseRules = mutableListOf<LicenseRule>()
 
     @Suppress("UNUSED") // This is intended to be used by rule implementations.
-    val uncuratedPkg: Package by lazy { ruleSet.ortResult.getUncuratedPackageOrProject(pkg.metadata.id)!! }
+    val uncuratedPkg: Package by lazy {
```

Comment:
Have you considered to disable the reporting of this type of issue entirely temporarily? 
...Or alternatively disabling that violation for each file it appears on file level? To me the suppress things seem to reduce readability.

Reply:
> Have you considered to disable the reporting of this type of issue entirely temporarily?

Yes, and I decided against it. I want to highlight the exact places where these unsafe calls are used in order to more easily find them and address them one-by one going forward.

> To me the suppress things seem to reduce readability.

I partly agree, the code looks "uglier" now. But that's deliberate, to have an incentive to clean it up via a better implementation.

Reply:
So, would you be ok to move these at the minimum to the function level? (In my view the effect on readability is not problematic then anymore. The scope should still be small then, if not then like the cause is too large function.